### PR TITLE
Faq inventory updates

### PR DESCRIPTION
**Pull request by AndreaFilyo**
Added new section for Inventory API.

Content additions: Updates to FAQs for Inventory API - input from recent doc project review and API support feedback.

### CODE REVIEW

File: faq.adoc [hunk 1]
```diff
@@ -151,9 +154,59 @@ Clarification examples:
...
+****
+====
+.*How do I handle frequent inventory updates for Zettle integrations with a web shop?*
+[%collapsible]
+====
+The Pusher API lets you manage frequent updates by subscribing to specific events triggered by Zettle. Events can for example be changes to product library or inventories. When an event is triggered, a request with event information will be sent in real time. See https://github.com/iZettle/api-documentation/blob/master/pusher-api/overview.md[Pusher API].
+****
+====
 == Finance
 .*How do I get sum of all payments, fees, refunds, invoices?*
```
Comment (Astro-Choco) [inline faq.adoc:208]: , and invoices?

Did you mean to get a sum of all those the mentioned transaction types in one response?

**Review by Astro-Choco — COMMENTED** [review]
Looks good. :) Only minor comments.

File: faq.adoc [hunk 2]
```diff
@@ -2,7 +2,10 @@
 
 
 ====
-If you want to apply to receive access to our APIs, sign up for an account at https://developer.zettle.com/register[Zettle Developer Portal].
+To create apps and work with integrations, sign up for a Zettle developer account at the https://developer.zettle.com/register[Zettle Developer Portal]. To learn more about the different Zettle APIs, see the https://github.com/iZettle/api-documentation/blob/faq-inventory-updates/README.md[developer documentation].
```
[the remark this answers is not in the document]
Comment (AndreaFilyo) [reply faq.adoc:5]: Fixed

File: faq.adoc [hunk 3]
```diff
@@ -151,9 +154,59 @@ Clarification examples:
 ****
 ====
 
+== Inventory
+.*What does an integration with Inventory typically include?*
```
[the remark this answers is not in the document]
Comment (AndreaFilyo) [reply faq.adoc:158]: Hmm, left as-is, I think there is a slight difference in the two questions.

File: faq.adoc [hunk 4]
```diff
@@ -151,9 +154,59 @@ Clarification examples:
...
+[%collapsible]
+====
+****
+An integration between the Inventory service and an e-commerce system ensures that products and inventory levels are automatically synchronized between the platforms. Integrated merchants can manage in-store and online inventories from one place. They can also compare sales and manage updates for all products. 
+****
+====
+.*Which e-commerce integrations are available for Zettle?*
```
[the remark this answers is not in the document]
Comment (AndreaFilyo) [reply faq.adoc:165]: Fixed.

File: faq.adoc [hunk 5]
```diff
@@ -151,9 +154,59 @@ Clarification examples:
...
+====
+****
+The Inventory service lets you start tracking of products through the API. After creating a product through the Product Library API, you can immediately start tracking it using the `startTrackingProduct` parameter in `POST /organizations/{organizationUuid}/inventory`. See https://github.com/iZettle/api-documentation/blob/master/inventory-api/user-guides/manage-inventory-tracking/enable-tracking.md[Enable tracking].
+****
+====
+.*How do I automate inventory updates between Zettle and my 3rd-party web shop?*
+[%collapsible]
+====
+****
+There are pre-built integrations available for a number of e-commerce platforms, see https://www.zettle.com/gb/integrations[e-commerce integrations]. You can also build your own e- integration using the Product Library and Inventory APIs. This will synchronize inventory updates between Zettle and your web shop.
```
[the remark this answers is not in the document]
Comment (AndreaFilyo) [reply faq.adoc:198]: Fixed

File: faq.adoc (hunk 1 above)
Comment (AndreaFilyo) [reply faq.adoc:208]: Fixed. I think the answer was more of a high level suggestion how to approach the use case.

File: faq.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (AndreaFilyo) [reply faq.adoc]: Actually, you're right, it may be confusing, changed it to your suggestion.

**Review by Astro-Choco — APPROVED** [review]
Looks great😊

**Pull request merged**